vllm/env_override.py

```diff
@@ -3,17 +3,23 @@
 
 import torch
 
+from vllm.logger import init_logger
+
+logger = init_logger(__name__)
+
 # set some common config/environment variables that should be set
 # for all processes created by vllm and all processes
 # that interact with vllm workers.
 # they are executed whenever `import vllm` is called.
 
-if not os.path.exists('/dev/nvidia-caps-imex-channels'):
-    # normally, we disable NCCL_CUMEM_ENABLE because it
-    # will cost 1~2 GiB GPU memory with cudagraph+allreduce,
-    # see https://github.com/NVIDIA/nccl/issues/1234
-    # for more details.
-    # However, NCCL requires NCCL_CUMEM_ENABLE to work with
+if os.environ.get('NCCL_CUMEM_ENABLE') == '1':
+    logger.warning(
+        "NCCL_CUMEM_ENABLE explicitly set to 1, skipping override. "
+        "This may increase memory overhead with cudagraph+allreduce: "
+        "https://github.com/NVIDIA/nccl/issues/1234")
+elif not os.path.exists('/dev/nvidia-caps-imex-channels'):
+    logger.warning("Overriding NCCL_CUMEM_ENABLE=0")
```

Comment:
```suggestion
    logger.warning("Skipping NCCL_CUMEM_ENABLE=0 override for NVL72 system, this will increase memory usage.")
```

Reply:
Here we're not skipping the override because it's a non-NVL72 system. Removed this anyways